gunicorn/app/base.py

```diff
@@ -5,13 +5,15 @@
 import os
 import sys
 import traceback
+import types
+from importlib.machinery import SourceFileLoader
```

Comment:
Nit: Can you import it as `import importlib.machinery` and alphabetize all imports?